INVENTORY/create-tb-json.py

```
@@ -207,6 +207,93 @@ def scp_to_node(self, local_file, node_name, remote_path="~", ssh_user="core"):
+        # Call core-auth-key.sh (needs KUBECONFIG from self.kubeconfig)
+        install_cmd = ["bash", key_script, node_name, pub_key]
+        
+        # Need to set KUBECONFIG environment for the script
+        if self.kubeconfig:
+            import os as os_module
+            env = os_module.environ.copy()
+            env["KUBECONFIG"] = self.kubeconfig
+            # Use subprocess directly with env
+            import subprocess
+            try:
+                result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)